egui_glow/src/painter.rs

```diff
@@ -61,18 +61,33 @@ impl Painter {
     /// Create painter.
     ///
     /// Set `pp_fb_extent` to the framebuffer size to enable `sRGB` support on OpenGL ES and WebGL.
+    /// Set `workaround` if you want to turn on shader workaround e.g. `&vec!["#define EPIPHANY_WORKAROUND".to_owned()]`.
+    ///
+    /// this fix [Everything is super dark in epiphany](https://github.com/emilk/egui/issues/794)
     /// # Errors
     /// will return `Err` below cases
     /// * failed to compile shader
     /// * failed to create postprocess on webgl with `sRGB` support
     /// * failed to create buffer
-    pub fn new(gl: &glow::Context, pp_fb_extent: Option<[i32; 2]>) -> Result<Painter, String> {
+    pub fn new(
+        gl: &glow::Context,
+        pp_fb_extent: Option<[i32; 2]>,
+        workarounds: &[String],
```

Comment:
I think just calling this `shader_prefix: &str` here and elsewhere would be clearer